### ISSUE TITLE
"Seeing What Happens" quest cannot be completed

### ISSUE THREAD

**Issue by Zmitrok1984** [slowtorta/turtlewow-bug-tracker#1076]
Quest **Seeing What Happens**
Horde: https://database.turtle-wow.org/?quest=2966
Alliance: https://database.turtle-wow.org/?quest=2946
can not completed. 

**Steps to reproduce the behavior with as much detail as possible:**
1. Take quest in Ironforge/Thundebluff
2. Go to the Tanaris , coordinates 37.7, 81.5
3.  According to wowhead here is should be interactable object [Uldum Pedestal.](https://classic.wowhead.com/object=142343)
4. [Object](https://database.turtle-wow.org/?object=142343) is here, but it isn't clickable.
![image](https://user-images.githubusercontent.com/99164611/152747463-ce3504ae-0ddb-479d-9f53-2f15e63ada86.png)

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
Object: https://database.turtle-wow.org/?object=142343

**Expected behavior. Describe how it should work:**
It should be possible to complete current quest via clicking object Uldum Pedestal and start next quests in chain:
[The Stone Watcher](https://database.turtle-wow.org/?quest=2954) 
[Return to Ironforge](https://database.turtle-wow.org/?quest=2977) or [Return to Thunder Bluff](https://database.turtle-wow.org/?quest=2967)

**Comment by geojak** [comment]
Hello torta, I just (5 minutes ago) tried this quest and it is not completely resolved. 

I was able to interact with the stone and hand in the quest "see what happens". I then interacted with the ghost to accept the follow up weust "the stone watcher"

My chat log said that I accepted the quest and I was able to do the dialog options, and I got the objective completed message show up, but.... The quest is nowhere to be found inside my quest log and and I ant hand it in nor can I pick up the follow up quest "return to ironfroge"

I made an ingame ticket because I thoughts is a character issue, but I will cancel thaz ticket since it seems a server problem.
